src/Trixi.jl

```diff
@@ -213,6 +214,12 @@ function __init__()
   @require Plots="91a5bcdd-55d7-5caf-9e0b-520d859cae80" begin
     using .Plots: plot, plot!, savefig
   end
+
+  @require Makie="ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a" begin
+    include("visualization/makie_visualization.jl")
+    using .Makie: Makie, Attributes, mesh, mesh!, plot!, @recipe, lines!, @lift
```

Comment:
I'd prefer to import Plots routines and just have `using .Makie: Makie`. This would make it so that users could still use the Plots.jl recipes and type `plot(sol)` instead of `Plots.plot(sol)`. Does that sound OK?

Reply:
This stuff is just inside Trixi.jl. Thus, users will always be able to do something like
```julia
using Plots
plot(sol)
```
Restricting the using statement only for Makie would also be okay with me, but it might be a bit more clear for people reading the visualization code when we just prefix the few plotting commands there with `Plots.`.

Reply:
Ah, I see. Agreed, especially if we can overload `plot` in Makie as well.